en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md

```diff
@@ -71,120 +82,182 @@ Environment compatibility
+-   The `config-validation.xml` file in the `<PRODUCT_HOME>/repository/conf/etc` directory contains a list of 
+recommended system parameters, which are validated against your system when the server starts. See [Configuring 
+config-validation.xml](https://docs.wso2.com/display/ADMIN44x/Configuring+config-validation.xml) for details on 
+modifying these parameters before starting the server.
+-   The Management Console uses the default [HTTP-NIO transport](https://docs.wso2
+.com/display/ADMIN44x/HTTP-NIO+Transport), of which the configurations are reflected in the `catalina-server.xml` 
+file in the `<PRODUCT_HOME>/repository/conf/tomcat` directory. According to the new depolyment.toml based configuration model, to
+ change the configurations in this file, please add the below configurations with appropriate values in 
+ `<PRODUCT_HOME>/repository/conf/deployment.toml` file.
+ 
+??? Note "Click to view the configurations"
+    ````
+    [transport.http.properties]
+    port = 9763
+    redirectPort = 9443
+    bindOnInit = false
+    maxHttpHeaderSize = 8192
+    acceptorThreadCount = 2
+    maxThreads = 250
+    minSpareThreads = 50
+    disableUploadTimeout = false
+    connectionUploadTimeout = 120000
+    maxKeepAliveRequests = 200
+    acceptCount = 200
+    server = "WSO2 Carbon Server"
+    compression = "on"
+    compressionMinSize = 2048
+    noCompressionUserAgents = "gozilla, traviata"
+    compressibleMimeType = "text/html,text/javascript,application/x-javascript,application/javascript,application/xml,text/css,application/xslt+xml,text/xsl,image/gif,image/pg,image/jpeg"
+    URIEncoding = "UTF-8"
+    
+    [transport.https.properties]
+    port = 9443
+    bindOnInit = false
+    maxHttpHeaderSize = 8192
+    acceptorThreadCount = 2
+    maxThreads = 250
+    minSpareThreads = 50
+    disableUploadTimeout = false
+    enableLookups = false
+    connectionUploadTimeout = 120000
+    maxKeepAliveRequests = 200
+    acceptCount = 200
+    server = "WSO2 Carbon Server"
+    compression = "on"
+    scheme = "https"
+    secure = true
+    SSLEnabled = true
+    compressionMinSize = 2048
+    noCompressionUserAgents = "gozilla, traviata"
+    compressibleMimeType = "text/html,text/javascript,application/x-javascript,application/javascript,application/xml,text/css,application/xslt+xml,text/xsl,image/gif,image/jpg,image/jpeg"
+    URIEncoding = "UTF-8"
+    
+    [transport.https.sslHostConfig.properties]
+    protocols = "+TLSv1,+TLSv1.1,+TLSv1.2"
+    sslProtocol = "TLS"
+    certificateVerification = false
+    ````
+ 
+These configurations has to be set properly for the management console to be accessible.
+
+-   As explained in the [installation prerequisites](#installation-prerequisites) , the default product installation uses OpenJDK. Therefore, you don't require a different JDK. However, if you have set up Oracle JDK or IBM JDK, be sure to apply the following settings to your product distribution.
```

Comment:
Therefore, you don't require a **separate** JDK

Reply:
Fixed with 238139a755fcb1391f32be7af75e89190fd5c965